integreat_cms/cms/templates/regions/region_form.html

```diff
@@ -128,6 +128,12 @@ <h3 class="heading font-bold text-black"><i data-feather="star" class="mr-2"></i
                         <label for="{{ form.locations_enabled.id_for_label }}">{{ form.locations_enabled.label }}</label>
                         <div class="help-text">{{ form.locations_enabled.help_text }}</div>
                     </div>
+                    <div>
+                        <!-- Options related to Fallback language -->
+                        {% render_field form.fallback_translations_enabled %}
+                        <label for="{{ form.fallback_translations_enabled.id_for_label }}">{{ form.fallback_translations_enabled.label }}</label>
```

Comment:
Could you add some text-ellipsis to this label. Otherwise, I have some strange display due to the length of the text.
![grafik](https://user-images.githubusercontent.com/15608678/170276004-7b0745fd-312d-4b07-9330-d3e5cba0f6b4.png)